net.technicpack.TechnicLauncher.metainfo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<component type="desktop">
+    <id>net.technicpack.TechnicLauncher</id>
+    <metadata_license>MIT</metadata_license>
+    <name>Technic Launcher</name>
+    <summary>The Technic Launcher will get you playing your favorite modpacks in no time. Choose from thousands of modpacks and install them with a single click.</summary>
+    <description>
+    	<p>Stylish, Technic, The launcher you all remember, Technic Launcher.</p>
+        <p>Features:</p>
+        <ul>
+          <li>
+          Easy Modpack Installs
+          
+          It's as easy as clicking play. The Technic Launcher handles all of the heavy lifting for you. You will be playing your favorite modpack in minutes.
+          </li>
+          
+          <li>
+          Discover
+          
+          Explore the Technic Platform from right within the launcher. Find the most popular modpacks of the week and install them right then and there. Just like that you are playing an all new modpack!
+          </li>
+        </ul>
+    </description>
+   
+    <categories>
+        <category>Game</category>
+    </categories>
+    
+    <url type="homepage">https://www.technicpack.net</url>
+    <url type="bugtracker">https://forums.technicpack.net/</url>
+    <url type="help">https://www.technicpack.net/help</url>
+    <url type="contact">https://www.technicpack.net/contact-us</url>
+
+    <launchable type="desktop-id">net.technicpack.TechnicLauncher.desktop</launchable>
+
+    <project_license>GPL-3.0</project_license>
```

Comment:
I don't see any indication that this is the license of the software on the upstream download page.

Reply:
Its hidden in the jar file.

They seem to be hiding it

Reply:
@hfiguiere 
https://github.com/TechnicPack/LauncherV3/blob/master/LICENSE
![Screenshot from 2022-01-25 18-55-03](https://user-images.githubusercontent.com/38189170/151079508-6c5b3871-e049-40ea-a365-b8717ec1b8bc.png)
![Screenshot from 2022-01-25 18-54-58](https://user-images.githubusercontent.com/38189170/151079509-c6c2ccf9-f7a5-43e4-a557-94bc95972a57.png)

Reply:
The core seems to be LGPL, While the launcher on git is GPLv3

Reply:
![Screenshot from 2022-01-25 19-00-17](https://user-images.githubusercontent.com/38189170/151081745-f35ab2d3-d10d-446f-8724-3778bd247e65.png)

Reply:
ok